sheets/1/exercise-3.tex

```diff
@@ -11,160 +11,161 @@
+	public <RückgabeTyp> <OperationsName>(<Typ> <ParameterName>, ...){
 		// Inhalt der Operation
 	}
 \end{lstlisting}
 
 \begin{lstlisting}[title=\textbf{Operations Beispiel}]
-	void springen(){
+	public void springen(){
 		// Code um zu springen
 	}
-
-	Hund sucheEinenHundAus(String wunschFarbe){
+	
+	public Hund sucheEinenHundAus(String wunschFarbe){
 		// Code um einen Hund auszusuchen
 	}
 \end{lstlisting}
 
- \begin{Infobox}
-              Operationen erfüllen zwei Aufgaben. Man kann sie dazu verwenden um Informationen von einem Objekt zu bekommen.
-              In dem Fall werden sie auch Abfragen (oder engl. Queries) genannt.
-              Oder man verwendet sie um etwas zu tun. In dem Fall werden sie Kommandos (oder engl. commands) gennant.
+\begin{Infobox}
+	
+	\subsubsection*{Objekte erzeugen:}
+	Objekte sind Instanzen von Klassen.
+	Um ein Objekt zu erzeugen wird das \lstinline{new} Schlüsselwort verwendet.
+	Man schreibt dann \lstinline{new } gefolgt von dem KlassenNamen und ein paar Klammern ().
 \end{Infobox}
-
-\subsubsection*{Objekte erzeugen:}
-Objekte sind Instanzen von Klassen.
-Um ein Objekt zu erzeugen wird das \lstinline{new} Schlüsselwort verwendet.
-Man schreibt dann \lstinline{new } gefolgt von dem KlassenNamen und ein paar Klammern ().
-
 \begin{lstlisting}[title=\textbf{Objekt erstellen Syntax}]
-new <KlassenName>(<MöglicheParameter>);// Ein Objekt wird erzeugt
+	new <KlassenName>(<MöglicheParameter>);// Ein Objekt wird erzeugt
 \end{lstlisting}
 
 \begin{lstlisting}[title=\textbf{Objekt erstellen Beispiel}]
-new Hund("Border Collie");
+	new Hund("Border Collie");
 \end{lstlisting}
-
-\subsubsection*{Variablen:}
-Variablen werden genutzt um Werte zu speichern.
-Sie sind nützlich in Fällen wo ihr einen bestimmten Wert mehrfach im selben Code nutzen wollt, oder wenn ein Wert nicht fest ist sondern während dem Ausführen bestimmt wird.\\
-Variablen besitzen immer einen Typ und einen Namen.
-Mit einem = kann einer Variable ein Wert zugewiesen werden.
-Wenn man ein Objekt erzeugt wird es meist direkt einer Variable zugewiesen, da man es später noch nutzen möchte.
-
+\begin{Infobox}
+	\subsubsection*{Variablen:}
+	Variablen werden genutzt um Werte zu speichern.
+	Sie sind nützlich in Fällen wo ihr einen bestimmten Wert mehrfach im selben Code nutzen wollt, oder wenn ein Wert nicht fest ist sondern während dem Ausführen bestimmt wird.\\
+	Variablen besitzen immer einen Typ und einen Namen.
+	Mit einem = kann einer Variable ein Wert zugewiesen werden.
+	Wenn man ein Objekt erzeugt wird es meist direkt einer Variable zugewiesen, da man es später noch nutzen möchte.
+\end{Infobox}
 \begin{lstlisting}[title=\textbf{Variablen Syntax}]
-	<KlassenName> <Name> // Eine Variable wird erzeugt, ihr ist noch kein Wert zugewiesen.
-
-	<KlassenName> <Name> = <Wert> // Eine variable wird erzeugt und ihr wird ein Wert zugewiesen.
+	public <KlassenName> <Name>; // Eine Variable wird erzeugt, ihr ist noch kein Wert zugewiesen.
```

Comment:
Variablen in Methoden lönnen kein public haben! (public sollte nur vor Klassen und Methoden stehen)